client/components/CandidateReview/CandidateTestPlanRun/InstructionsRenderer.jsx

```diff
@@ -169,19 +170,128 @@ const InstructionsRenderer = ({
         commandsContent
     );
 
-    const assertions = [...pageContent.instructions.assertions.assertions];
-    const assertionsContent = parseListContent(assertions);
-
+    headingLevel = 5;
     const Heading = `h${headingLevel}`;
 
     return (
         <>
             <NumberedList>{allInstructionsContent}</NumberedList>
-            <Heading>{pageContent.instructions.assertions.header}</Heading>
-            {pageContent.instructions.assertions.description}
-            <NumberedList>{assertionsContent}</NumberedList>
             {settingsContent.length ? settingsContent : null}
 
+            {renderableContent.commands.map(({ id, settings }, i) => {
+                // debugger;
+                const settingsScreenText = isV2
+                    ? renderableContent.target.at.raw.settings[settings]
+                          ?.screenText ?? ''
+                    : null;
+
+                let mustCount = 0;
+                let shouldCount = 0;
+                let mayCount = 0;
+
+                renderableContent.assertions.forEach(({ priority }) => {
+                    const priorityString = convertAssertionPriority(priority);
+                    if (priorityString === 'MUST') mustCount += 1;
+                    if (priorityString === 'SHOULD') shouldCount += 1;
+                    if (priorityString === 'MAY') mayCount += 1;
+                });
+
+                const settingsScreenTextFormatted =
+                    settingsScreenText === null
+                        ? ''
+                        : ` (${settingsScreenText})`;
+
+                const scenarioTitle =
+                    `${renderableContent.commands[i].keystroke}` +
+                    `${settingsScreenTextFormatted}: ${mustCount} MUST, ` +
+                    `${shouldCount} SHOULD, ${mayCount} MAY Assertions`;
+
+                if (isV2) {
+                    return (
+                        <>
```

Comment:
The console is reporting that a unique key prop is required here. React uses this to determine if an element needs to be re-rendered. To have a unique enough key here, and based on the context, you could do something like:
```javascript
<React.Fragment key={`command-${id}-${i}`}>
```